k8s/environments/ithc/common-overlay/video-hearings/vh-aad-identity.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: vh-aad-identity
+ namespace: vh
+spec:
+  type: 0
+  resourceID: /subscriptions/ba71a911-e0d6-4776-a1a6-079af1df7139/resourceGroups/managed-identities-ithc-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-ithc-kvuser
```

Comment:
can you confirm theses
Subscription and ClientID

Reply:
These are correct.